packages/light-client-extension-helpers/src/background/background-helper.ts

```diff
@@ -40,6 +41,8 @@ export const register = ({
   if (isRegistered) throw new Error("helper already registered")
   isRegistered = true
 
+  supervise(smoldotClient)
```

Comment:
To get a better understanding, why doesn't `supervise` need an `await`?

Reply:
register isnt marked as async (maybe it should be)? so it is indeed just a dangling promise here